README.md

```diff
@@ -158,6 +158,7 @@ If you use the server-side flow, Facebook will give you back a longer lived acce
 
 Actively tested with the following Ruby versions:
 
+- MRI 2.2.0
 - MRI 2.1.0
```

Comment:
Seems right.
Currently this version is incorrect.

But patch version in README is hardly to maintain.
Might it be `2.1` or `2.1.x` ?